proto/pdpb.proto

```diff
@@ -168,4 +178,9 @@ message BootstrappedError {
 message Error {
     optional string message                 = 1;
     optional BootstrappedError bootstrapped = 2;
+}
+
+message Job {
```

Comment:
For every conf change, split operation, we will add this into job list, and in another thread, get first job, handle it, and then pop.  I will add a comment here.